### ISSUE TITLE
emacs -nw cannot Enter or Backspace

### ISSUE THREAD

**Issue by luigy** [akermu/emacs-libvterm#67]
When I hit `Enter`, `Backspace` or move the trackpad I only get unexpected output in vterm under `emacs -nw` (works as expected only on gui)
```bash
# how I ran the cmd in the clip
TERM=xterm emacs -nw -Q -L .emacs.d/.local/packages/elpa/vterm-20190219.1719/ --eval "(require 'vterm)" --eval "(vterm)"
```
![vterm-emacs-nw](https://cldup.com/Phw5vrQjst.gif)
tested this with bash,sh,fish and urxvt/termite and saw the same behavior
any thought how I can debug this further?

**Comment by luigy** [comment]
As a workaround I'm using
```emacs-lisp
  (unless (display-graphic-p)
    (define-key vterm-mode-map [up]    '(lambda () (interactive) (vterm-send-key "<up>")))
    (define-key vterm-mode-map [down]  '(lambda () (interactive) (vterm-send-key "<down>")))
    (define-key vterm-mode-map [right] '(lambda () (interactive) (vterm-send-key "<right>")))
    (define-key vterm-mode-map [left]  '(lambda () (interactive) (vterm-send-key "<left>")))
    (define-key vterm-mode-map [tab]   '(lambda () (interactive) (vterm-send-key "<tab>")))
    (define-key vterm-mode-map (kbd "DEL") '(lambda () (interactive) (vterm-send-key "<backspace>")))
    (define-key vterm-mode-map (kbd "RET") '(lambda () (interactive) (vterm-send-key "<return>"))))

```

**Comment by akermu** [comment]
This is solved by #108.
